samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -98,6 +105,12 @@ private TestRunner() {
     configs.put(JobConfig.PROCESSOR_ID(), "1");
     configs.put(JobCoordinatorConfig.JOB_COORDINATOR_FACTORY, PassthroughJobCoordinatorFactory.class.getName());
     configs.put(TaskConfig.GROUPER_FACTORY(), SingleContainerGrouperFactory.class.getName());
+    // RockDb Tables if used needs this to be configured if no changelog is configured
```

Comment:
s/RockDb Tables if used needs this to be configured if no changelog is configured/Setup rocksDB stores without changelog.